packages/module/CHANGELOG.md

```diff
@@ -1,5303 +0,0 @@
-# Change Log
```

Comment:
Should the changelog have been deleted?

And followup question, does this have any/all changes to the log viewer package based on the most recent release from patternfly-react? Looks like the most recent release in this old changelog was from September.

Reply:
no